helpers/aws.js

```diff
@@ -58,6 +58,41 @@ class AwsClient {
             });
         });
     }
+
+    /**
+     * Delete all cached objects at cachePath
+     * @method invalidateCache
+     * @param {String}              cachePath       Path to cache
+     * @param {Function}            callback        callback function
+     */
+    invalidateCache(cachePath, callback) {
+        const self = this;
+
+        let params = {
+            Bucket: this.bucket,
+            Prefix: `caches/${cachePath}`
+        };
+
+        return this.client.listObjects(params, (e, data) => {
+            if (e) return callback(e);
+
+            if (data.isTruncated) return callback();
```

Comment:
what is this check for?

Reply:
The response is paginated. The check is for when it's the last page of results.

Reply:
if it's the last page, shouldn't it still do one more delete?